src/tools/rustfmt/src/visitor.rs

```diff
@@ -592,6 +592,7 @@ impl<'b, 'a: 'b> FmtVisitor<'a> {
                     );
                     self.push_rewrite(item.span, rewrite);
                 }
+                ast::ItemKind::Delegation(..) => {}
```

Comment:
Might want to change this so rustfmt doesn't totally remove the Delegation item.

```suggestion
                ast::ItemKind::Delegation(..) => {
                    // TODO: rewrite delegation items once syntax is established.
                    // For now, leave the contents of the Span unformatted.
                    self.push_rewrite(item.span, None)
                }
```